### PR TITLE
[docs] policies/accessibility: Migrate Accessibility docs

### PR DESCRIPTION
**Pull request by andrewnicols**
Migrated from:

- https://docs.moodle.org/dev/Accessibility

### CODE REVIEW

File: general/development/policies/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+---
+title: Accessibility
+tags: []
```
Comment (sarjona) [inline general/development/policies/accessibility.md:3]:
```suggestion
tags:
  - Accessibility
```

File: general/development/policies/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+Key pages within Moodle, which are representative of its overall accessibility and functionality, are audited using both automated tools and user journey testing. The WCAG 2.1 Level A and Level AA issues raised from this audit were all addressed in Moodle 3.10 and Moodle 3.9.3 (both released on November 9th 2020). Many of these [accessibility improvements](https://tracker.moodle.org/browse/MDL-67688) where also back-ported to Moodle 3.7.5+ and Moodle 3.8.2+, where possible.
+
+:::info Moodle's accreditations
+
+Both Moodle, and the Moodle Mobile App hold a WCAG 2.1 AA accreditation based on the successful resolution of issues identified during audits.
+
+These accreditations are renewed each year.
+
+<div className={ clsx( 'container' ) } >
```
Comment (sarjona) [inline general/development/policies/accessibility.md:22]: What's the benefit of using clsx here? As there are no conditionals, why you haven't used class="container" (which doesn't require importing clsx)?

**Review by sarjona — CHANGES_REQUESTED** [review]
Thanks for working on this page, Andrew. It looks great and, you'll see, I have only a couple of comments :-)

**Comment by andrewnicols** [comment]
Sorted those. I added some additional tags too.

File: general/development/policies/accessibility.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+
+:::info Moodle's accreditations
+
+Both Moodle, and the Moodle Mobile App hold a WCAG 2.1 AA accreditation based on the successful resolution of issues identified during audits.
+
+These accreditations are renewed each year.
+
+<div className='container' >
+    <div className='row' >
+        <div className='col col--6' >
```
Comment (sarjona) [inline general/development/policies/accessibility.md:25]:
```suggestion
        <div className='col col-6' >
```

File: general/development/policies/accessibility.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+            <figure>
+                <a href="https://www.webkeyit.com/accessibility-services/accessibility-accreditations/moodle">
+                    <img width="400px" src="https://www.webkeyit.com/accessibility-services/accessibility-accreditations/moodle/badge" alt="WCAG 2.1 AA Accreditation for Moodle LMS" />
+                    <figcaption>Moodle WCAG 2.1 AA Accreditation, first issues November 10th 2020; Re-accredited on November 8th 2021</figcaption>
+                </a>
+            </figure>
+        </div>
+        <div className='col col--6' >
```
Comment (sarjona) [inline general/development/policies/accessibility.md:33]:
```suggestion
        <div className='col col-6' >
```

**Review by sarjona — CHANGES_REQUESTED** [review]
Thanks Andrew! One more thing I've spotted while doing the last check! :-)

**Comment by andrewnicols** [comment]
I'm told that the class names come from http://getbem.com/naming/

**Review by sarjona — APPROVED** [review]
Hi Andrew! Ok, got it! I haven't seen classNames before but now I understand better how they work (thanks for the link!).

**Comment by andrewnicols** [comment]
Since Docusaurus is built with React, this is useful: https://reactjs.org/docs/faq-styling.html

**Pull request merged**